tooling/api/src/main/java/org/jboss/windup/tooling/IProgressMonitorAdapter.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.windup.tooling;
+
+import java.util.logging.LogRecord;
+
+public interface IProgressMonitorAdapter 
```

Comment:
:) long time not seeing this naming convention for Interfaces :smile:

Reply:
Old habit :-)